Makefile

```diff
@@ -213,13 +213,7 @@ kill_all_port_forwardings:
 # Day 2 #
 #########
 
-deploy_on_ocp_cluster: bring_assisted_installer
-	# controller
-	DEPLOY_TARGET=ocp NAMESPACE_INDEX=$(shell bash scripts/utils.sh get_namespace_index $(NAMESPACE)) \
-		DEPLOY_TAG=$(DEPLOY_TAG) DEPLOY_MANIFEST_TAG=$(DEPLOY_MANIFEST_TAG) OCP_KUBECONFIG=$(OCP_KUBECONFIG) \
-		PROFILE=$(PROFILE) CONTROLLER_OCP=$(CONTROLLER_OCP) \
-		scripts/deploy_controller.sh
-
+deploy_on_ocp_cluster:
```

Comment:
why it is now a different target? Why can't it be part of deploy_on_ocp_cluster

Reply:
The controller needs the ID of the new cluster, so separated into two stages:
1. deploy_on_ocp_cluster
2. deploy_controller_on_ocp_cluster CLUSTER_ID=123
I.e. The cluster_id is available after the cluster is registered in step 1.

Reply:
Fetched the cluster id in deploy_controller.sh, so no need to manually specify it now.